go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/google/go-cmp v0.5.6
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/viper v1.8.1
-	github.com/vektah/gqlparser/v2 v2.2.0
+	github.com/vektah/gqlparser/v2 v2.1.0
```

Comment:
I had to downgrade to resolve https://stackoverflow.com/questions/67125011/gqlgen-directiveresolver-not-exported-by-package-generated